src/dialogs/FileAndPayInvalidNameRequestDialog.vue

```diff
@@ -1,7 +1,15 @@
 <template>
-  <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="file-pay-nr-error-dialog">
+  <v-dialog
+    v-model="dialog"
+    width="45rem"
+    persistent
+    :attach="attach"
+    content-class="file-pay-nr-error-dialog"
+  >
     <v-card>
-      <v-card-title id="dialog-title">Invalid Name Request (NR) / Incorporation Application</v-card-title>
+      <v-card-title id="dialog-title">
+        Invalid Name Request (NR) / Incorporation Application
+      </v-card-title>
       <v-card-text id="dialog-text">
```

Comment:
Since you're here, could you add a blank line above this (between title block and text block) please? Thanks!